Pinta/MainWindow.cs

```diff
@@ -386,10 +387,6 @@ private void CreateMainMenu ()
 		PintaCore.Actions.Edit.RegisterActions (app, edit_menu);
 		this.menu_bar.AppendSubmenu (Translations.GetString ("_Edit"), edit_menu);
 
-		var view_menu = Gio.Menu.New ();
```

Comment:
This shifted the ordering of the menus when using a menu bar - the view menu is now appearing after `Layers` instead of after `Edit`
For muscle memory I think we should keep the existing menu bar order

Reply:
My bad, I tested the presence of the View menu in the menubar, but forgot to test the order. Fixed.